app/models/concept.rb

```diff
@@ -0,0 +1,3 @@
+class Concept < ApplicationRecord
+  belongs_to :question
```

Comment:
Do we want to have a one to many relationship like this and not a many to many? I.e. each concept you create will only have one question it is linked to, aka two questions with "travel" as a concept would need separate "travel" concept objects - is this correct?